query-node/manifest.yml

```diff
@@ -0,0 +1,57 @@
+version: '0.1'
+description: Joystream query-node manifest file for sumer
+repository: https://github.com/Joystream/joystream
+dataSource:
+  kind: substrate 
+  chain: joystream
+  indexerVersion: '0.1.6'
+entities:
+  - mappings/lib/generated/**/*.model.js
+metadata:
+  # TODO: update source and blockHash for sumer release
+  source: wss://babylon-sub-1.joystream.app/staging/rpc
+  blockHash: '0x206a6cca31333cb19ee120cde7455e13eea368ec3f6176d0cd48698228410489'
```

Comment:
I don't think we should really need to specify `blockHash` here. By default, it just takes the latest block which should be fine